src/Nethermind/Nethermind.Merge.Plugin/IMergeConfig.cs

```diff
@@ -69,4 +69,7 @@ The number of requests to the garbage collector (GC) to release the process memo
 
     [ConfigItem(Description = "[TECHNICAL] Simulate block production for every possible slot. Just for stress-testing purposes.", DefaultValue = "false", HiddenFromDocs = true)]
     bool SimulateBlockProduction { get; set; }
+
+    [ConfigItem(Description = "Whether to show the block's Extra Data field in logs", DefaultValue = "true")]
+    bool ShowExtraData { get; set; }
```

Comment:
I think we don't need that as a param and just show it.

Reply:
Removed as param